### DISCUSSION TITLE
Components Tests

### DISCUSSION THREAD

**Discussion by TheDevick** [shadcn-ui/ui#5008] [Ideas]
Hey! 

First of all, thanks for this package. It's awesome!

I wanted to propose a feature that allows us to include test files when installing a component.
This subject was already mentioned twice but, but it hasn’t made much progress.

And also, one question: Where are the tests from the components? I tried searching in this repository but didn't found tests for the components itself.
I need that so I can see what you guys are testing in a specific component and maybe use it in my tests.

**Comment by pau-not-paul** [comment]
I think there are no component tests.
